### PR TITLE
Add type annotations to tests

### PR DESCRIPTION
**Pull request by dangquangdon**
Add type annotations to:
- `test_argx.py`
- `test_cli.py`
- `test_cliarg.py`
- `test_session.py`

[DARK-557]

[DARK-557]: https://aiven.atlassian.net/browse/DARK-557?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: tests/test_argx.py [hunk 1]
```diff
@@ -70,22 +71,22 @@ def test_extended_commands_remain_alphabetically_ordered():
 
 class DescriptorCLI(CommandLineTool):
     @property
-    def raise1(self):
+    def raise1(self) -> None:
```
Comment (rominf) [inline tests/test_argx.py:74]: It should return `NoReturn`.

Comment (rominf) [inline tests/test_argx.py:74]: It should return `NoReturn`.

File: tests/test_argx.py [hunk 2]
```diff
@@ -70,22 +71,22 @@ def test_extended_commands_remain_alphabetically_ordered():
...
         raise RuntimeError("evaluated raise1")
 
     if cached_property is not None:
 
         @cached_property
-        def raise2(self):
+        def raise2(self) -> None:
```
Comment (rominf) [inline tests/test_argx.py:80]: It should return `NoReturn`.

**Review by rominf — DISMISSED** [review]
Please resolve the conflicts and address the comments. Otherwise, LGTM.

**Review by dmitrii-vasilev — APPROVED** [review]
LGTM

**Pull request merged**